jose/encrypt.go

```diff
@@ -104,7 +104,7 @@ func Decrypt(data []byte, opts ...Option) ([]byte, error) {
 		if data, err = enc.Decrypt(ctx.password); err == nil {
 			return data, nil
 		}
-		return nil, errors.New("failed to decrypt JWE: invalid password")
+		return nil, errors.New("failed to decrypt JWE: invalid password" + err.Error())
```

Comment:
Added a space, but other options might work too. No `fmt.Errorf`?
```suggestion
		return nil, errors.New("failed to decrypt JWE: invalid password " + err.Error())
```